docs/tutorials/kubernetes-basics/expose-intro.html

```diff
@@ -12,126 +12,101 @@
+				<li><i>ExternalName</i> - Exposes the Service using an arbitrary name (specified by <code>externalName</code> in the spec) by returning a CNAME record with the name. No proxy is used. This type requires v1.7 or higher of <code>kube-dns</code>.</li>
+			</ul>
+			<p>More information about the different types of Services can be found in the <a href="/docs/tutorials/services/source-ip/">Using Source IP</a> tutorial. Also see <a href="/docs/concepts/services-networking/connect-applications-service">Connecting Applications with Services</a>.</p>
+			<p>Additionally, note that there are some use cases with Services that involve not defining <code>selector</code> in the spec. A Service created without <code>selector</code> will also not create the corresponding Endpoints object. This allows users to manually map a Service to specific endpoints. Another possibility why there may be no selector is you are strictly using <code>type: ExternalName</code>.</p>
+			</div>
+			<div class="col-md-4">
+				<div class="content__box content__box_lined">
+					<h3>Summary</h3>
+					<ul>
+						<li>Exposing Pods to external traffic</li>
+						<li>Load balancing traffic across multiple Pods</li>
+						<li>Using labels</li>
+					</ul>
+				</div>
+				<div class="content__box content__box_fill">
+						<p><i>A Kubernetes Service is an abstraction layer which defines a logical set of Pods and enables external traffic exposure, load balancing and service discovery for those Pods.</i></p>
+				</div>
+			</div>
+		</div>
+			<p></p>
+		</div>
+		<br>
+
+		<div class="row">
+			<div class="col-md-8">
+				<h3>Services and Labels</h3>
+			</div>
+		</div>
+
+		<div class="row">
+			<div class="col-md-8">
+				<p><img src="/docs/tutorials/kubernetes-basics/public/images/module_04_services.svg"></p>
+			</div>
+		</div>
+
+		<div class="row">
+			<div class="col-md-8">
+				<p>A Service routes traffic across a set of Pods. Services are the abstraction that allow pods to die and replicate in Kubernetes without impacting your application. Discovery and routing among dependent Pods (such as the frontend and backend components in an application) is handled by Kubernetes Services.</p>
+				<p>Services match a set of Pods using <a href="/docs/concepts/overview/working-with-objects/labels">labels and selectors</a>, a grouping primitive that allows logical operation on objects in Kubernetes. Labels are key/value pairs attached to objects and can be used in any number of ways:</p>
+				<ul>
+					<li>Designate objects for development, test, and production</li>
+					<li>Embed version tags</li>
+					<li>Classify an object using tags</li>
+				</ul>
+
+			</div>
+			<div class="col-md-4">
+				<div class="content__box content__box_fill">
+					<p><i>You can create a Service at the same time you create a Deployment by using `--expose` in kubectl.</i></p>
```

Comment:
Since markdown won't work here, you might want to remove the back ticks and try \<code>\</code>. Also, currently, "--expose" is being wrapped, so I recommend putting a newline before the double hyphen.

![screen shot 2017-04-03 at 11 00 57 am](https://cloud.githubusercontent.com/assets/24798125/24623614/dc4fd27a-185d-11e7-92bf-4d1630dd94b5.png)